src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -38,38 +35,42 @@ public void onAppStart(IApplicationInstance appInstance)
     	}
     }
 
-    
+    // Invoked when an HTTP MPEGDash Streaming session is created; 
+    // defined in IModuleOnHTTPMPEGDashStreamingSession module interfaces
     public void onHTTPMPEGDashStreamingSessionCreate(HTTPStreamerSessionMPEGDash httpSession)
     {
     	getLogger().info("Tommy: SulWowza onHTTPMPEGDashStreamingSessionCreate: " + httpSession.getSessionId());
     	authorizeSession(httpSession);
     }
-    
+
+    // Invoked when an HTTP Cupertino Streaming session is created; 
+    // defined in IModuleOnHTTPCupertinoStreamingSession interface
     public void onHTTPCupertinoStreamingSessionCreate(HTTPStreamerSessionCupertino httpSession)
     {
 		getLogger().info("Tommy: SulWowza onHTTPCupertinoStreamingSessionCreate: ");
 		authorizeSession(httpSession);
 	}
-    
+
     void authorizeSession(IHTTPStreamerSession httpSession)
     {
-    	String query = httpSession.getQueryStr();
-    	if(authorize(query))
-    		httpSession.acceptSession();
-    	else httpSession.rejectSession();
+		String queryStr = httpSession.getQueryStr();
+		if (authorize(queryStr))
+			httpSession.acceptSession();
+		else
+			httpSession.rejectSession();
     }
-    
-    boolean authorize(String query)
+
+    boolean authorize(String queryStr)
     {
-    	String authToken = getAuthToken(query);
-    	return validateAuthToken(authToken);
+		String authToken = getAuthToken(queryStr);
+		return validateAuthToken(authToken);
     }
-    
-    String getAuthToken(String parameters)
+
+    String getAuthToken(String queryStr)
     {
-    	if (parameters != null)
+		if (queryStr != null && queryStr.length() > 6)  // "token=" is 6 chars
     	{
-    		String[] parts = parameters.split("\\?");
+			String[] parts = queryStr.split("\\?");
```

Comment:
yeah, I've been noticing that and thinking about it.  It's actually tabs vs. spaces, but it should be uniform.  Was trying to figure out how to get it to be always spaces in Eclipse and haven't figured it out yet.